src/serviceWorker.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-console */
```

Comment:
assuming this is intended

Reply:
Yup, we don't enable the service worker anyways but I don't want to interfere w/ the boilerplate code